godot_project/chemical_structures/aromatics/4_bis_4-dimethylaminophenyl_methylene-2_5-cyclohexadien-1-iminium.mol

```diff
@@ -1,111 +1,113 @@
 4-[Bis(4-dimethylaminophenyl)methylene]-2,5-cyclohexadien-1-iminium
- OpenBabel07100913143D
```

Comment:
This has been downloaded from https://chem-file.sourceforge.net/ oredited manually by you? i found a little fishy the dissapearence of this line